scripts/devnet-curio/.env

```diff
@@ -0,0 +1,24 @@
+CURIO_IMAGE=curio/curio-dev:dev
+LOTUS_IMAGE=ghcr.io/chainsafe/lotus-devnet:2024-07-03-1f030c2
+FOREST_DATA_DIR=/forest_data
+LOTUS_DATA_DIR=/lotus_data
+CURIO_REPO_PATH=/var/lib/curio
+FIL_PROOFS_PARAMETER_CACHE=/var/tmp/filecoin-proof-parameters
+MINER_ACTOR_ADDRESS=t01000
+LOTUS_RPC_PORT=1234
+LOTUS_P2P_PORT=1235
+MINER_RPC_PORT=2345
+FOREST_RPC_PORT=3456
+FOREST_OFFLINE_RPC_PORT=3457
+F3_RPC_PORT=23456
+F3_FINALITY=10
+GENESIS_NETWORK_VERSION=18
+SHARK_HEIGHT=-10
+HYGGE_HEIGHT=-9
+LIGHTNING_HEIGHT=3
+THUNDER_HEIGHT=6
+WATERMELON_HEIGHT=9
+DRAGON_HEIGHT=12
+WAFFLE_HEIGHT=18
+DRAND_QUICKNET_HEIGHT=15
+TARGET_HEIGHT=24
```

Comment:
For the sake of simplicity, it might make sense to start directly from NV23 and avoid migrations.